lib/cdo/contact_rollups/v2/pardot.rb

```diff
@@ -113,7 +115,7 @@ def self.build_prospect_query_url(id_greater_than, fields, limit, deleted)
   def batch_create_prospects(email, data, eager_submit = false)
     prospect = self.class.convert_to_pardot_prospect data.merge(email: email)
     @new_prospects << prospect
-
+    # Creating new prospects is not a retriable action, unlike updating existing prospects.
```

Comment:
Why not?

Reply:
Is this better?
```suggestion
    # Creating new prospects is not a retriable action because it could succeed  
    # on the Pardot side and we just didn't receive a response. If we try again,
    # it would create duplicate prospects.
```